daemon/config.go

```diff
@@ -17,7 +17,7 @@ const (
 // common across platforms.
 type CommonConfig struct {
 	AutoRestart    bool
-	Bridge         bridge.Config
+	Bridge         bridgeConfig
```

Comment:
I know the surrounding fields are documented, but let's not continue that. Please document according to [this blog](http://blog.golang.org/godoc-documenting-go-code).